frontend/src/gamestate.js

```diff
@@ -210,12 +210,6 @@ class GameState extends EventEmitter {
     return this.#burnCardIds.includes(cardId.toString());
   }
 
-  // mixmix - what is this? it doesn't seem to be used for anything
-  isAutoremovableAutopick(cardId, picksPerPack) {
-    if (this.#pickCardIds.length == picksPerPack && picksPerPack > 1){
-      return this.#pickCardIds[0] == cardId.toString();
-    }
-  }
```

Comment:
@HerveH44 Can you please check this - I think this was safe to remove, but I didn't fully understand what it was intended to be helping do (nothing that I could see)

Reply:
I think it comes from another feature (multi autopicks) and this one was just checking if the cardId is the one that will be remove if I select another card. 
I am sure you can delete it. It must be a leftover of a refactor.

Reply:
How sorry, "I" am going to delete it. Didn't see the comment was on the main PR xD ! Pushing the delete ASAP

Reply:
Though it's still used by the CardDefault component. I think it should stay there unless you also want to change the behaviour of that Component @mixmix ?

Reply:
It's only in CardDefault because I refractored it there as I was doing this work and was wanting to preserve existing code.

The reason I think it can go is that it is displaying something (I still don't quite understand what), but it wasn't be used by any logic anywhere else to **do** anything.

So is this fine to remove?